neighbors/src/components/homePanel/HomePanel.jsx

```diff
@@ -0,0 +1,50 @@
+import './stylesHomePanel.css';
+
+export const HomePanel = () => {
+  const slides = [
```

Comment:
Zmienną `slides` możemy zdefiniować poza komponentem.
Ogólna zasada, jeżeli dana zmienna nie zależy od propsów komponentu (tak jak tutaj) to wydzielamy ją poza komponent, trzymajmy wewnątrz komponentu najmniej rzeczy jak się da.